File: gp_game/gener_level.js

var fs = require("fs");
function writeFile(filename, data){
    // 2、写文件
    fs.writeFile(filename, data, function (error) {
        if (error) {
            // 出现错误
        }
        // 继续操作
    });

}

function gener_level(){
    str = "";
    for(var i = 1; i <= 60; ++i){
        str += "level-" + i + ",关卡-" + i + "\n"
    }
    return str
}

writeFile("level.csv", gener_level())

File: gp_game/gener_level.py
#!/usr/bin/env python
# coding: utf-8

def writeFile(filename, content) :
    file = open(filename, 'w+')
    file.write(content)
    file.close()

def generator() :
	str = ""
	for( i = 1; i <= 60; ++i ) :
		str += "level-" + str(i) + ",关卡-" + str(i) + "\n"
	return str

 if __name__ == "__main__": 
 	str = generator()
 	writeFile("level.csv", str)